src/libstd/sys/windows/time.rs

```diff
@@ -12,7 +12,7 @@ use ops::Sub;
 use time::Duration;
 use sync::{Once, ONCE_INIT};
 
-const NANOS_PER_SEC: i64 = 1_000_000_000;
+const NANOS_PER_SEC: u64 = 1_000_000_000;
```

Comment:
Why that `NANOS_PER_SEC` constant became `u32` and this became `u64`? Can we deduplicate them so changing the constant when somebody makes `nano-` mean a different multiplier easier?